controllers/User/userController.js

```diff
@@ -0,0 +1,82 @@
+const validateUser = require('../../utils/validations/index')
+const bcrypt = require('bcryptjs')
+const User = require('../../models/User/userModel')
+const response = require('../../utils/libs/response')
+const Joi = require('joi')
+
+// Register Validation schema
+const registrationSchema = Joi.object({
+    firstName: Joi.string().required(), 
+    lastName: Joi.string().required(), 
+    email: Joi.string().email().required(),
+    password: Joi.string().required().min(8),
+    phone: Joi.number().required()
+})
+
```

Comment:
I am actually calling the functin to validate.. what I tried defining was the schema since it was where the req.body is coming from... also, different feature (login, signup, contact us, e.t.c) would need different schema.. is there a better way to handle it it considering the different req.body?